javascript/javascript1/week1/FutureAgeCalculator.js

```diff
@@ -0,0 +1,6 @@
+var yearOfBirth = 1988;
+var yearFuture = 2030;
+
+var yourAgeInFuture = yearFuture-yearOfBirth;
```

Comment:
try not to use `var` you always `const` and if you want to change the variable value use `let`
you can find many articles about why not to use `var` [here is one of them](https://blog.usejournal.com/awesome-javascript-no-more-var-working-title-999428999994)